app/resources/v1/pacbio/library_resource.rb

```diff
@@ -24,17 +27,26 @@ class LibraryResource < JSONAPI::Resource
 
       has_one :primary_aliquot, always_include_optional_linkage_data: true,
                                 relation_name: :primary_aliquot, class_name: 'Aliquot'
+      has_many :used_aliquots, always_include_optional_linkage_data: true,
+                               relation_name: :used_aliquots, class_name: 'Aliquot'
 
       paginator :paged
 
       def self.default_sort
         [{ field: 'created_at', direction: :desc }]
       end
 
-      def self.records(_options = {})
+      def self.records(options = {})
         # We only want to include only libraries without pools
         # Libraries with pools should be referenced via the LibraryPoolResource
-        super.where(pool: nil)
+        # This breaks aliquot polymorphism so we only filter if the resource
+        # Is accessed directly
+        if options[:include_directives]
```

Comment:
This isn't ideal because we added the LibraryPoolResource to avoid doing this. But this adds a filter to the aliquot polymorphism that wipes out pools and we can't change the type from 'libraries' to 'librarypools' so we have to do this.

Reply:
Will this go after we do the tidy up?

Reply:
Yes